.github/workflows/build-main-branches.yml

```diff
@@ -25,6 +25,14 @@ jobs:
                         'name': '',
                         'version': '17',
                     },
+                    {
+                        'name': '',
+                        'version': '20',
+                    },
+                    {
+                        'name': '',
+                        'version': '21-ea',
```

Comment:
It didn't work last week...

Reply:
The previous Jandex update was for this java 21.